modules/tsb/cp/main.tf

```diff
@@ -89,6 +89,7 @@ resource "helm_release" "controlplane" {
     ratelimit_enabled               = var.ratelimit_enabled
     ratelimit_namespace             = var.ratelimit_namespace
     identity_propagation_enabled    = var.identity_propagation_enabled
+    vm_endpoint                     = "vms.${var.cluster_name}.${regep("[^\\.]*\\.(.*)", var.tsb_fqdn)}"
```

Comment:
the regexp is basically removing the first block to capture the DNS suffix

Reply:
to keep it generic... should we have a vm_endpoint as a variable for tsb_cp module ... $var.tsb_fqdn may be or can be different from external-dns domain... so basically under module itself... we should define vm_endpoint as

vm_endpoint = $var.cluster_name + lookup/element to select external_dns_$cloud_dns_zone